<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Locar_Cadastro_Veiculo</class>
 <widget class="QMainWindow" name="Locar_Cadastro_Veiculo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Locar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(85, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="subtitulo_console">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWidth">
     <number>8</number>
    </property>
    <property name="midLineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string>Console de Gerenciamento Locar</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_selecione_opcao">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>180</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>45</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Selecione a opção</string>
    </property>
   </widget>
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Serviços Adicionais</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Quantidade de Diarias</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Seguro Veiculo Roubo e Furto</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Seguro Veicular Completo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Seguro Veicular Acidente</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Forma de Pagamento</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Veiculo Escolhido pelo cliente</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Bebê Conforto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Data Devolução</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_12">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Condutor Extra</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Cadeira de Bebê</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Data Retirada</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Limpeza Completa</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_13">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>460</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Limpeza Simples</string>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>CPF do Cliente</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label7_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>620</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string>Valor Total:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label7_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>620</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
border-color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
   <widget class="QComboBox" name="opcoes_combo_box">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>230</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>230</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>Cadastrar Veiculo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cadastrar Reserva</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cadastrar Cliente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Consultar Veiculo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Consultar Reserva</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Consultar Cliente</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
